cub/agent/single_pass_scan_operators.cuh

```diff
@@ -127,20 +169,20 @@ struct ScanTileState<T, true>
     // Status word type
     using StatusWord = cub::detail::conditional_t<
       sizeof(T) == 8,
-      long long,
+      unsigned long long,
```

Comment:
std::uint64_t ?

Reply:
Will address in a separate [issue](https://github.com/NVIDIA/cub/issues/623#issue-1574406833)